cookbooks/aws-parallelcluster-config/recipes/base.rb

```diff
@@ -27,16 +27,23 @@
 
 include_recipe 'aws-parallelcluster-config::nfs' unless virtualized?
 
-service "setup-ephemeral" do
-  supports restart: false
-  action :enable
-end
+# Disable mounting of ephemeral drive under /scratch if /scratch is used by another device
```

Comment:
can you please rephrase this comment, PR title and description to reflect the fact that we aren't considering only the "/scratch" value?